calculator/main.py

```diff
@@ -26,7 +28,23 @@ def perform_operation(calc, operation, params):
     :param params: Tuple containing the list of nums to operate with.
                    ie: (1, 2, 3, 4.5, -2)
     """
-    pass
+    if operation not in get_operations(calc):
```

Comment:
I like this use of the abstraction. Can you also remember to talk about this on Saturday? I'd like to share it with the other groups.